scripts/compare_benchmarks.py

```diff
@@ -12,10 +14,16 @@
 min_runtime_ns = 59 * 1000 * 1000 * 1000
 
 def format_diff(diff):
-    if diff >= 0:
-        return colored('+' + "{0:.0%}".format(diff), 'green')
-    else:
+    if diff < 0:
         return colored("{0:.0%}".format(diff), 'red')
+    else:
+        return colored("+{0:.0%}".format(diff), 'green')
+
+def format_change(change):
+    if change < 0:
+        return colored("▼ {:.4f}".format(abs(change)), 'red')
```

Comment:
u"..."

Reply:
Not required for python3 and not helping for python2. I'll take a look at this issue again when we have agreed on what to do with this PR.

Reply:
But I have it working with that? Did you also fix the magic comment?